flexget/tests/api_tests/test_database_api.py

```diff
@@ -37,5 +37,5 @@ def test_database_get_plugins(self, api_client, schema_match):
         assert rsp.status_code == 200
         data = json.loads(rsp.get_data(as_text=True))
 
-        errors = schema_match(OC.plugin_list, data)
+        errors = schema_match(ObjectsContainer.plugin_list, data)
```

Comment:
What rule is this even? About not being camelcase? I'd argue OC is camelcase.

Reply:
It is [N817](https://docs.astral.sh/ruff/rules/camelcase-imported-as-acronym/) `camelcase-imported-as-acronym`. Ruff considers fully uppercase `OC` to be potentially mistaken for a constant by readers. If you don't like this rule, I will disable it.

Reply:
I think we can disable this rule.